build/source/driver/multi_driver.f90

```diff
@@ -423,6 +424,9 @@ program multi_driver
 ! get the number of HRUs in the run domain
 nHRUrun = sum(gru_struc%hruCount)
 
+! define the chunksize (for netcdf read/write)
+chunksize=8*nHRUrun
```

Comment:
I suspect that ~2x nHRUrun would be better, but you'd have to run some tests on this.  The files themselves should probably have larger chunk sizes to accommodate runs with larger numbers of HRUs, but once you know exactly how many HRUs you have you might even do better setting `chunksize = nHRUrun`.  The writes you might want to keep larger so that other applications working with more data at once will work well, though if you are writing one file / process, then `chunksize = nHRUrun` might be better there too.

Reply:
This is 2x nHRUrun -- since most of the forcing data is stored as 4-byte reals.

Reply:
I should really have `8` be a parameter